tests/README.md

```diff
@@ -0,0 +1,26 @@
+## How to get started
+
+Python 3.6 is required.
+For settting up Python 3 environment, [virtualenv](https://virtualenv.pypa.io/en/stable/) is commonly used, like:
```

Comment:
I personal opinion about python environment manager is that basically standard `venv` is the first option and if you want to use various python version then `pyenv` + `pyenv+virtualenv` is good.

Reply:
Anyway, I don't want to explain the python environment management here so how about just recommending readers to make their own environment somehow.

Reply:
Yes, it is just recommendation, so removing it is OK.
Some other projects like [py-evm](
https://github.com/ethereum/py-evm#development) explain about venv, then I added it.

(I also use `pyenv` + `pyenv+virtualenv` :) )